src/libraries/Native/Unix/System.Native/pal_interfaceaddresses.h

```diff
@@ -38,7 +38,7 @@ typedef struct
 {
     char Name[16];              // OS Interface name.
     uint32_t InterfaceIndex;    // Interface index.
-    int32_t Speed;              // Link speed for physical interfaces.
+    int64_t Speed;              // Link speed for physical interfaces.
```

Comment:
This struct is no longer efficiently packed; can we move the `Speed` member up above `InterfaceIndex`?

Reply:
Could this be changed to `uint32_t `instead?

Reply:
Done, thanks for noticing!